checkout-fees-for-woocommerce/includes/class-alg-wc-checkout-fees.php

```diff
@@ -296,7 +296,10 @@ public function add_gateways_fees( $the_cart ) {
 			if ( ! $current_gateway ) {
 				return;
 			}
-
+			$klarna_payment = 'klarna_payments';
+			if ( strpos( $current_gateway, $klarna_payment ) !== false ) {
+				$current_gateway = 'klarna_payments';
+			}
```

Comment:
Please add a comment here on why we needed to add this piece of code for Klarna gateway.